### ISSUE TITLE
Application name should be inherited from folder name when dot is used

### ISSUE THREAD

**Issue by onnimonni** [tongueroo/jets#42]
If I create a new project into existing folder:
```bash
$ mkdir jets-demo
$ cd jets-demo
$ jets new .
```

It works otherwise but the: `config/application.rb` ends up looking like this:
```ruby
Jets.application.configure do
  config.project_name = "."
  ...
end
```

And the dot results in name error with AWS when I try to deploy it.

**Comment by tongueroo** [comment]
Ah, nice interface. Yes, this should be added. Tagging issue with help wanted

**Comment by tongueroo** [comment]
Addressed in #46 Will go out in the next release

**Comment by tongueroo** [comment]
Released in v0.10.1 [CHANGELOG](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md)
